frontend/src/hooks/useLogin.ts

```diff
@@ -31,9 +31,19 @@ export const useLogin: LoginHook = () => {
       setResult({ ...result, isLoading: false });
       const { data, error } = await client.auth.signInWithPassword(credentials);
       setResult({ ...result, data, error });
```

Comment:
Can you also fix one thing we forget when adding these hooks? set `isLoading` to true before we call supabase and set it to false after we get the result. Same for the `useSignUp` hook. That way I can close the PR I made.

Reply:
Done